ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/components/StatusCountIcon.tsx

```diff
@@ -18,11 +18,12 @@ type StatusCountIconProps = {
     text: string;
     status: Status;
     count: number;
+    disabled?: boolean;
```

Comment:
A question for your thoughts, not a change request.

What are pro and con of `disabled` as optional? Pardon pun on `?`

Reply:
I thought of `StatusCountIcon` as more of a general reusable compliance component, where the disabled prop might not always be necessary, especially when filtering isn’t involved. I know it’s best not to code for uncertain future needs, but that was my thought process.